install/kubernetes/helm/istio/charts/mixer/templates/service.yaml

```diff
@@ -1,31 +1,30 @@
+{{ $mixers := list "check" "report" }}
+{{- range $idx, $mname := $mixers }}
 apiVersion: v1
 kind: Service
 metadata:
-  name: istio-mixer
+  name: istio-mixer-{{ $mname }}
   labels:
-    app: {{ template "mixer.name" . }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    chart: {{ $.Chart.Name }}-{{ $.Chart.Version | replace "+" "_" }}
+    release: {{ $.Release.Name }}
     istio: mixer
 spec:
   ports:
   - name: tcp-plain
     port: 9091
   - name: tcp-mtls
     port: 15004
-  - name: http-health
+  - name: http-monitoring
     port: 9093
-  - name: configapi
-    port: 9094
-  - name: statsd-prom
-    port: 9102
-  - name: statsd-udp
-    port: 9125
-    protocol: UDP
+{{- if eq $mname "report" }}
```

Comment:
for your consideration: when want to have mixer monitored by mixer, it _might_ make sense to allow Report() calls locally (mixer would implement istio-policy, but also provide metrics export)....

Reply:
I think Mixer can always call mixer locally even when the port is not exposed.
It also also call `istio-telemetry`